app/components/login/LoginComponent.jsx

```diff
@@ -73,7 +72,7 @@ class LoginComponent extends React.Component {
 
     handleLoginError = () => {
         const errorText = <span>Invalid Username/Password, please try again</span>
-        ReactDOM.render(errorText, document.getElementById('login-error'));
+        _render(errorText, document.getElementById('login-error'));
```

Comment:
Is there a reason why you are using the ReactDOM render here instead of just returning html to the render function and display that? The is not breaking anything but does not follow the logic of rest of application and can be confusing to future developers. Please let me know if you will be able to update the code to html logic.